documentation/tutorials/StateAssimilation/TreeRingSDA.Rmd

```diff
@@ -40,48 +40,66 @@ Perform a site-level SIPNET run using the following settings
 * At the top of the file add the following tags to set the ensemble size
 
 ```
-<state.data.assimilation>
-  <n.ensemble>35</n.ensemble>
-  <process.variance>FALSE</process.variance>
-  <sample.parameters>TRUE</sample.parameters>
+   <state.data.assimilation>
+   <n.ensemble>35</n.ensemble>
+   <process.variance>FALSE</process.variance>
+   <sample.parameters>TRUE</sample.parameters>
+   <data>
+    <format_id>
+    <format_id>1000000040</format_id>
+    </format_id>
```

Comment:
1) If we have input.id, we shouldn't need the user to specify format_id as well.
2) we definitely shouldn't need to nest format_id within format_id